src/lib.rs

```diff
@@ -1,3 +1,33 @@
+//! Library with common functionality for I2C based sensors from Sensirion, based on the
+//! `embedded-hal` traits.
+//!
+//! ## Usage
+//!
+//! ### CRC8 calculation / validation
```

Comment:
Depending on the markdown dialect, you might need an empty line between the heading and the code block. (I think it's OK for rustdoc though.)

Reply:
Fixed anyways :slightly_smiling_face: